### PR TITLE
Remove "I have performed a self-review of my changes" from the pull request template

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
# Description

We don't have a CLA but the guidelines should be help enough.

- [x] I have read and understood the [contributing guidelines](/erlef/setup-beam/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by paulo-ferraz-oliveira** [comment]
Removing the "I have performed..." item from the template, as per our recent discussion, since the content seems to potentially be mis-understood.

File: .github/workflows/ubuntu.yml [hunk 1]
```diff
@@ -55,7 +55,7 @@ jobs:
             os: 'ubuntu-22.04'
           - otp-version: '24'
             elixir-version: '1.16'
-            rebar3-version: '3'
+            rebar3-version: '3.23'
```
Comment (paulo-ferraz-oliveira) [inline .github/workflows/ubuntu.yml:58]: It automatically chose 3.24 so I'm pinning this down to have CI pass.

**Review by wojtekmach — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
Thanks; merging...

**Comment by paulo-ferraz-oliveira** [comment]
Released in v1, v1.18 and v1.18.2. Thanks.

**Pull request merged**